docs/Workflow.md

```diff
@@ -1,32 +1,55 @@
 <!--- SPDX-License-Identifier: Apache-2.0 -->
 # Contribution Guide
 
-## Prerequisite: Install LLVM
+## Step 1: Fork ONNX-MLIR in the cloud
 
-Follow directions in this section of the top level [README](../README.md#mlir) 
-for installing the currently supported LLVM version. We assume here a
-LINUX installation. 
+We strongly encourage contributors to work in their own forks of the ONNX-MLIR project.
+Creating a new fork is easy.
 
-### Step 1: Fork in the cloud
-
-1. Visit https://github.com/onnx/onnx-mlir
+1. Visit https://github.com/onnx/onnx-mlir.
 2. Click `Fork` button (top right) to establish a fork.
 
-### Step 2: Clone fork to local storage
+For the rest of this guide, we assume that you have set `user` to match your github profile name:
+
+```sh
+user={your github profile name}
+```
```

Comment:
3. Navigate to your newly created fork, click on the the green `Code` button to get the link to *your* newly-created ONNX-MLIR fork:
```
git@github.com:<username>/onnx-mlir.git
```

where `<username>` is your GitHub username.